src/interfaces/ICSBenefitsRegistry.sol

```diff
@@ -0,0 +1,86 @@
+// SPDX-FileCopyrightText: 2024 Lido <info@lido.fi>
+// SPDX-License-Identifier: GPL-3.0
+
+pragma solidity 0.8.24;
+
+interface ICSBenefitsRegistry {
+    event RewardShareDataSet(uint256 indexed curveId);
+    event PerformanceLeewayDataSet(uint256 indexed curveId);
+    event PriorityQueueLimitSet(uint256 indexed curveId, uint256 limit);
+
+    error InvalidRewardShareData();
+    error InvalidPerformanceLeewayData();
+    error InvalidPriorityQueueLimit();
+    error NoData();
+    error ZeroAdminAddress();
+
+    /// @notice Set reward share parameters for the curveId
+    /// @dev keyPivots = [10, 50] and rewardShares = [10000, 8000, 5000] stands for
+    ///      100% rewards for the keys 1-10, 80% rewards fot the keys 11-50, and 50% rewards for the keys > 50
+    /// @param curveId Curve Id to associate reward share data with
+    /// @param keyPivots Pivot numbers of the keys (ex. [10, 50])
+    /// @param rewardShares Reward share percentages in BP (ex. [10000, 8000, 5000])
+    function setRewardShareData(
+        uint256 curveId,
+        uint256[] calldata keyPivots,
+        uint256[] calldata rewardShares
+    ) external;
+
+    /// @notice Get reward share parameters by the curveId.
+    /// @dev Reverts if the values are not set for the given curveId.
+    /// @dev keyPivots = [10, 50] and rewardShares = [10000, 8000, 5000] stands for
+    ///      100% rewards for the keys 1-10, 80% rewards fot the keys 11-50, and 50% rewards for the keys > 50
+    /// @param curveId Curve Id to get reward share data for
+    /// @return keyPivots Pivot numbers of the keys (ex. [10, 50])
+    /// @return rewardShares Reward share percentages in BP (ex. [10000, 8000, 5000])
+    function getRewardShareData(
```

Comment:
from offchain perspective would be useful to have smth like
```
function getRewardShareByKeysCount(
    uint256 curveId, uint256 keysCount
) external view returns (uint16 rewardSharesBP);
```
and the same for performance leeway